src/variant_detection/variant_output.cpp

```diff
@@ -66,58 +68,57 @@ void write_record(Cluster const & cluster,
     record.filter() = {"PASS"};
     record.genotypes() = {};
     record.genotypes().push_back({ .id = "GT", .value = std::vector{"./."s}});
+    record.info() = {};
 
     if (mate1.orientation == mate2.orientation)
     {
         if (mate1.seq_name == mate2.seq_name)
         {
-            size_t insert_size = cluster.get_average_inserted_sequence_size();
             if (mate1.orientation == strand::forward)
             {
```

Comment:
These could be merged into one condition

Reply:
You mean the two if cases?
I opened this PR because I actually want to call INVs. For calling INVs there will be an `else` for `if (mate1.orientation == mate2.orientation)`. So I would leave this as it is.

Reply:
The three nested if cases actually... 
Of course, if you add else branches later, then it won't be possible.